modules/analysis-common/src/main/java/org/elasticsearch/analysis/common/MultiplexerTokenFilterFactory.java

```diff
@@ -62,6 +62,11 @@ public MultiplexerTokenFilterFactory(IndexSettings indexSettings, Environment en
         this.preserveOriginal = settings.getAsBoolean("preserve_original", true);
     }
 
+    @Override
+    public TokenFilterFactory getSynonymFilter() {
+        return null;
```

Comment:
should it be null or a dummy factory that doesn't wrap the token stream? I'm afraid that we might be exposing ourselves to null-pointer exceptions?